answers.md

```diff
@@ -0,0 +1,102 @@
+# mssw Chapter 9 Answers
+
+## 1. What class is at the top of the exception hierarchy?
+
+`Throwable`
+
+## 2. Briefly explain how to use try and catch
+
+You would wrap a `try` block around some code that you would expect to throw an exception.
+
+After the `try` block you would add a `catch` block to stop any specified exceptions halting the execution of your program. 
+
+## 3. What is wrong with this fragment?
+
+```java
+// ...
+vals[18] = 10;
+catch (ArrayIndexOutOfBoundsException exc) {
+	// handle error
+} 
+```
+
+The comment is stating that it is handling an error, when in fact it is handling an exception.
```

Comment:
It states _handle error_ with a lowercase error, i.e. not the `Error` class. It's valid to refer to catching exceptions as a means of error handling.